multinet/api/tests/conftest.py

```diff
@@ -92,6 +105,8 @@ def pytest_sessionfinish(session, exitstatus):
 
 
 register(UserFactory)
-register(WorkspaceFactory)
+register(PrivateWorkspaceFactory, _name='workspace')
+register(PublicWorkspaceFactory, _name='public_workspace')
```

Comment:
`_name` defaults to the lowercase-underscore model name, so in this case I can remove the `_name='workspace'` from the `PrivateWorkspaceFactory` registration. I agree with eventually renaming that fixture to `private_workspace`. I think when it comes time to parametrize all of these tests it would be good to incorporate this change, if parametrized tests still use similar fixtures. (issue #56 )